dynamicEDT3D/package.xml

```diff
@@ -1,25 +1,27 @@
-<package>
+<package format="3">
   <name>dynamic_edt_3d</name>
   <version>1.9.0</version>
   <description> The dynamicEDT3D library implements an inrementally updatable Euclidean distance transform (EDT) in 3D. It comes with a wrapper to use the OctoMap 3D representation and hooks into the change detection of the OctoMap library to propagate changes to the EDT.</description>
 
   <author email="sprunkc@informatik.uni-freiburg.de">Christoph Sprunk</author>
+
+  <maintainer email="anasarrak@erlerobotics.com">Anas M'chichou</maintainer>
   <maintainer email="sprunkc@informatik.uni-freiburg.de">Christoph Sprunk</maintainer>
   <license>BSD</license>
 
   <url type="website">http://octomap.github.io</url>
   <url type="bugtracker">https://github.com/OctoMap/octomap/issues</url>
 
-  <build_depend>octomap</build_depend>
 
-  <run_depend>octomap</run_depend>  
-  <run_depend>catkin</run_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <buildtool_depend>cmake</buildtool_depend>
+  <build_depend>octomap</build_depend>
+
+  <exec_depend>octomap</exec_depend>
+  <!-- <exec_depend>catkin</exec_depend> -->
 
-  
   <export>
-    <build_type>cmake</build_type>
+    <build_type>ament_cmake</build_type>
```

Comment:
With this, I expect that you do not have to modify the CMakeLists, but I may be mistaken. Can you please clarify?